### ISSUE TITLE
Error passing style tag via withSize()

### ISSUE THREAD

**Issue by chrism2671** [reactivemarkets/react-financial-charts#398]
Hi Mark,

I've never used Typescript, so am learning how it works, while retrofitting this into a React app. Great work by the way.

It seems like it's mostly there, except I'm getting the following error:

![2020-07-19-181846_1050x306_scrot](https://user-images.githubusercontent.com/456541/87880938-a91e7880-c9ed-11ea-8cfe-c0b514e35ea3.png)

The offending line is:
`export default withSize({ style: { minHeight: 600 } })(withDeviceRatio()(StockChart));
`

I'm 99.9% sure the syntax is correct, and the types are legit too. Any idea what I'm doing wrong?

**Comment by markmcdowell** [comment]
Hey, it doesn't look immediately wrong. Which version of typescript and which version of this library are you using? I'll see if I can reproduce.

**Comment by chrism2671** [comment]
Thanks! It's probably just my lack of experience with TS, but here's the env in any case:

```
    "react-financial-charts": "^1.0.0-alpha.2",
    "awesome-typescript-loader": "^5.2.1",
    "source-map-loader": "^1.0.1",
    "ts-loader": "^8.0.1",
    "typescript": "^3.9.7"

```
tsconfig.json:
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react"
  },
  "include": [
    "src"
  ]
}

```

**Comment by markmcdowell** [comment]
Yeah weird one, you need to add types dependency for the auto sizer `npm i -D @types/react-virtualized-auto-sizer` the typescript compiler can't work out the properties are optional.

I'll see if I can work around it in the package but installing the types is a quick fix.

**Comment by markmcdowell** [comment]
Fixed in [v1.0.0-alpha.4](https://github.com/reactivemarkets/react-financial-charts/releases/tag/v1.0.0-alpha.4)

**Comment by chrism2671** [comment]
Amazing, thanks Mark!!! :)
